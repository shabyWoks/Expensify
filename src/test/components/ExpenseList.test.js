import React from 'react';
import {shallow} from 'enzyme';
import {ExpenseList} from '../../components/ExpenseList';
import {expenses} from '../fixtures/expenses';

test('should render expense list when expenses is passed', () => {
    const wrapper = shallow(<ExpenseList expenses={expenses} />);
    expect(wrapper).toMatchSnapshot();
});

test('should render expense list when expenses is not passed', () => {
    const wrapper = shallow(<ExpenseList expenses={[]} />);
    expect(wrapper).toMatchSnapshot();
});