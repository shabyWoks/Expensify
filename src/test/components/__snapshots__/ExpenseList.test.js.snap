// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense list when expenses is not passed 1`] = `
<div>
  <h1>
    Expense List
  </h1>
  <p>
    No Expense
  </p>
</div>
`;

exports[`should render expense list when expenses is passed 1`] = `
<div>
  <h1>
    Expense List
  </h1>
  <ExpenseListItem
    amount={195}
    createdAt={0}
    description="Gur"
    id="1"
    key="1"
    note=""
  />
  <ExpenseListItem
    amount={10950}
    createdAt={-345600000}
    description="Rent"
    id="2"
    key="2"
    note=""
  />
  <ExpenseListItem
    amount={4500}
    createdAt={259200000}
    description="Credit Card"
    id="3"
    key="3"
    note=""
  />
</div>
`;
